tests/linearGaussian_snpe_test.py

```diff
@@ -41,12 +41,11 @@
 )
 
 
-@pytest.mark.parametrize("npe_method", [NPE_A, NPE_C])
```

Comment:
makes sense to remove `npe_c` here as we test it below with different density estimators, good catch!